pkgs/development/interpreters/ruby/default.nix

```
@@ -59,6 +48,13 @@ let
       , buildEnv, bundler, bundix
       , libiconv, libobjc, libunwind, Foundation
       , makeWrapper, buildRubyGem, defaultGemConfig
+      , baseRuby ? if stdenv.hostPlatform != stdenv.buildPlatform
+        then buildPackages.ruby
+        else if useRailsExpress then self.override {
+          useRailsExpress = false;
+          docSupport = false;
+          rubygemsSupport = false;
+        } else null
```

```suggestion
      , baseRuby ? buildPackages.ruby.override {
          useRailsExpress = false;
          docSupport = false;
          rubygemsSupport = false;
        }
      , useBaseRuby ? stdenv.buildPlatform != stdenv.hostPlatform || useRailsExpress
```

and then useBaseRuby as the condition below instead of (baseRuby != null)

---

I like the unconditional use of `buildPackages.ruby` here, thanks!

@SuperSandro2000 done!